### PR TITLE
OEP-2: Formatting Updates

### PR DESCRIPTION
**Pull request by nasthagiri**
This PR simply reformats OEP-2 according to the latest standards.  There are no content changes included.

### CODE REVIEW

File: oeps/oep-0002-bp-repo-metadata.rst [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
-+---------------+-------------------------------------------+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-2 <oep-0002-bp-repo-metadata>`
+   * - Title
+     - Repository Metadata
+   * - Last Modified
+     - 2019-11-14
```
Comment (jmbowman) [inline oeps/oep-0002-bp-repo-metadata.rst]: Minor point - should this be updated to 11/22 now?

File: oeps/oep-0002-bp-repo-metadata.rst [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
+   * - Authors
+     - Calen Pennington, Feanil Patel, Nimisha Asthagiri
```
Comment (jmbowman) [inline oeps/oep-0002-bp-repo-metadata.rst]: Are we deliberately shifting away from including email addresses?  OEP-1 just mentions them as optional, but still has it as required for the arbiter.

File: oeps/oep-0002-bp-repo-metadata.rst (hunk 1 above)
Comment (nasthagiri) [reply oeps/oep-0002-bp-repo-metadata.rst]: I can go either way on this.  Do reformatting changes require updating the timestamp?  I'm also not sending out reformatting changes for a wider review.  Git already catches these iterative timestamps, right?

File: oeps/oep-0002-bp-repo-metadata.rst (hunk 2 above)
Comment (nasthagiri) [reply oeps/oep-0002-bp-repo-metadata.rst]: Good catch.  I'll add the email address for the arbiter.

I was influenced by the fact that later OEPs didn't include email addresses.  (Side note: going through and updating the formatting of all OEPs may be a one-time worthwhile investment.)

**Pull request merged**